AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -401,6 +401,57 @@ private MouseMode CurrentMode
         /// </summary>
         public List<LayoutObject> SelectedObjects { get; set; }
 
+        /// <summary>
+        /// Add the objects to SelectedObjects, optionally also add all objects which match one of their identifiers.
+        /// </summary>
+        private void AddSelectedObjects(List<LayoutObject> objectsToAdd, bool includeSameObjects)
```

Comment:
How would you feel about renaming `includeSameObjects` to `includeObjectsWithMatchingIndentifier`? The downside is that it is a little on the long side, but it gets across the intention of the parameter. Alternatively, we could add `<param name="includeSameObjects"> description of parameter </param>` under the summary tag.

Either change would need to be propagated to each one of the methods that has an `includeSameObjects` parameter.

Reply:
I've gone down the documentation route.